<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuchungenAnschauen</class>
 <widget class="QWidget" name="BuchungenAnschauen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="HauptLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 118, 114);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>1241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 31, 52);
font: 600 italic 15pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Buchungen Anschauen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZurueck">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: #ffffff;
    font: 600 italic 15pt &quot;Segoe UI&quot;;
    padding: 10px 20px;
    border: 2px solid #2980b9;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f618d;
}
</string>
   </property>
   <property name="text">
    <string>Zur√ºck</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1341</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>600 11 25pt Segoe UI</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
        background-color: #f0f0f0;
        color: rgb(0, 0, 0);
    selection-background-color: #4286f4;
    selection-color: #ffffff;
    border: 1px solid #aaaaaa;
    font-family: 600 italic 25pt Segoe UI;
}

QTableView::item {
    padding: 5px;
    border-bottom: 1px solid #ddd;
}

QTableView::item:selected {
        background-color: #CDDC39;
    color: white;
}

QHeaderView::section {
        background-color: #4CAF50;
    color: white;
}</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonEinnahmen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {

        background-color: rgb(186, 90, 92);
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
        background-color: rgb(186, 90, 92);
}</string>
   </property>
   <property name="text">
    <string>Einnahmen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRechnungen">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>120</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {

        background-color: rgb(186, 90, 92);
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
        background-color: rgb(186, 90, 92);
}</string>
   </property>
   <property name="text">
    <string>Rechnungen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAusgaben">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton {
    padding: 5px;
    border: 2px solid #555;
    border-radius: 8px;
    background-color: #f2f2f2;
    color: #333;
}


QRadioButton:checked {

        background-color: rgb(186, 90, 92);
    color: white;
}


QRadioButton::indicator {
    width: 15px;
    height: 15px;
    border-radius: 7px;
    background-color: #ccc;
}


QRadioButton::indicator:checked {
        background-color: rgb(186, 90, 92);
}</string>
   </property>
   <property name="text">
    <string>Ausgaben</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
